ZFSin/zfs/module/zfs/zfs_windows_zvol.c

```diff
@@ -892,7 +892,7 @@ wzvol_StopAdapter(__in pHW_HBA_EXT pHBAExt)               // Adapter device-obje
 #if defined(_AMD64_)
 		KeReleaseInStackQueuedSpinLock(&LockHandle);
 #else
-		KeReleaseSpinLock(&pMPDrvInfo->MPIOExtLock, SaveIrql);
+		KeReleaseSpinLock(&pwzvolDrvInfo->MPIOExtLock, SaveIrql);
```

Comment:
Been tempted to remove the _AMD64_ here, as it appears to be just "more efficient", and the standard lock will work fine. It makes it harder to read, for practically no reason (only run when stopping).  Or is there some other reason to use `KeReleaseInStackQueuedSpinLock()` variants?